### PR TITLE
Add all ports to be forwarded in the devcontainer

### PR DESCRIPTION
**Pull request by dkatzz**
### Description

This adds the remaining ports to the devcontainer, as well as descriptions for each of them.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)

### Issue(s) this completes
https://github.com/civiform/civiform/issues/6999

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -70,5 +70,7 @@
   "postCreateCommand": "echo 'source /usr/share/bash-completion/completions/git' >> ~/.bashrc",
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [3390, 5432, 9000, 9999]
+  // 3390 is for OIDC, 4566 is for localstack (email), 5432 is for postgres, 6645 is for localstack (file uploads), 8888 is for the prod simulator
+  // 8888 is for the prod simulator, 9000 is for the application, 9100 is for unit tests, 9999 is for browser tests, 10000 is for azureite
+  "forwardPorts": [3390, 4566, 5432, 6645, 8888, 9000, 9100, 9999, 10000]
```
[the remark this answers is not in the document]
Comment (dkatzz) [reply .devcontainer/devcontainer.json]: Yeah good call, thanks - I actually decided to use [portsAttributes](https://containers.dev/implementors/json_reference/#port-attributes) instead. Let me know what you think of that.

[the remark this answers is not in the document]
Comment (jeffpw-goog) [reply .devcontainer/devcontainer.json]: Sounds good to me!

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -69,6 +69,15 @@
...
+  // Use 'forwardPorts' to make a list of ports inside the container available locally. The ports correspond to the following:
+  //   * 3390 - OIDC
+  //   * 4566 - localstack (email)
+  //   * 5432 - postgres
+  //   * 6645 - localstack (file uploads)
+  //   * 8888 - prod simulator
+  //   * 9000 - application
+  //   * 9100 - unit tests
+  //   * 9999 - browser tests
+  //   * 10000 - azureite
```
Comment (gwendolyngoetz) [inline .devcontainer/devcontainer.json]:
```suggestion
  //   * 3390 - OIDC (dev)
  //   * 4566 - localstack (dev email & file upload)
  //   * 5432 - postgres (dev)
  //   * 6645 - localstack (browser-test email & file upload)
  //   * 8888 - prod simulator
  //   * 9000 - application (dev)
  //   * 9100 - application (unit tests)
  //   * 9999 - application (browser tests)
  //   * 10000 - azureite
```

Comment (gwendolyngoetz) [reply .devcontainer/devcontainer.json]: Adding more details to the descriptions

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -69,6 +69,17 @@
   "updateContentCommand": "bin/vscode-setup",
   "postCreateCommand": "echo 'source /usr/share/bash-completion/completions/git' >> ~/.bashrc",
 
-  // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [3390, 5432, 9000, 9999]
+  // Use 'forwardPorts' to make a list of ports inside the container available locally. The ports correspond to the following:
```
Comment (jeffpw-goog) [inline .devcontainer/devcontainer.json:72]: Is this sentence stray? Not sure it makes sense with the new format?

**Review by jeffpw-goog — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 3 above)
Comment (dkatzz) [reply .devcontainer/devcontainer.json]: Yep you're right - thanks for catching!

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (dkatzz) [reply .devcontainer/devcontainer.json]: Thank you! Updated - but put it in the portAttributes format.

**Review by gwendolyngoetz — APPROVED** [review]

**Review by jeffpw-goog — APPROVED** [review]

**Pull request merged**